plutus-tx-plugin/test/Plugin/Data/recursive/interListConstruct.tplc-read.golden

```diff
@@ -0,0 +1,265 @@
+(program 1.1.0 ((/\(InterList_i0 :: * -> * -> *) ->
+                   \(InterCons_i0
+                       : all a_i0
+                             b_i0.
```

Comment:
should we have a space before the dot, perhaps?

Reply:
I don't like that, but YMMV.